### DISCUSSION TITLE
Integrazione Alexa media player

### DISCUSSION THREAD

**Discussion by paolomorabito** [indomus/forum#3229] [HUB: Home Assistant]
Ho seguito il tutorial su https://indomus.it/guide/integrare-amazon-alexa-come-media-player-su-home-assistant/ per integrare Alexa come media player. Ho attivato l'accesso a 2 passaggi, ho creato l'app di autenticazione, inserito il codice etc etc... Arrivo alla pagina di accesso ad amazon

![image](https://github.com/user-attachments/assets/e5309617-3d02-4271-a478-aa946f74403d)

Mi chiede la password. La inserisco e mi ritorna di nuovo alla pagina di accesso amazon (quella nell'immagine). 
Ho provato con diversi browser, ma niente...

Coma potrebbe essere?

**Comment by KC-inDomus** [comment]
clicca su creare nuovo account e prosegui; quando ti dirà che esiste già, clicca su sign in e vedrai che andrà avanti

**Reply by paolomorabito** [reply]
Assurdo! Ma funziona! Grazie
